design/FY2021/NFP-CodeCompliantEquipmentDataset.md

```diff
@@ -0,0 +1,65 @@
+Code Compliant Equipment Dataset
+================
+
+Jeremy Lerond, Aowabin Rahman, Jian Zhang, Reid Hart, Michael Rosenberg, PNNL
+
+ - 09/07/2021 
+
+## Justification for New Feature ##
+EnergyPlus requires modelers to select sets of performance curves to describe the part load performance of modeled HVAC equipment. While full load efficiency metrics can often be used as a direct input in models, performance curves dictate what the part load efficiency metric of a modeled piece of equipment is at different part load operating conditions.
+
+HVAC equipment rarely, if ever, operate exactly at their full load rated conditions so it is really important to select performance curves that appropriately describe their part load performance. When modeling an existing piece of equipment this can be done by generating performance curves using published manufacturer expanded performance tables. Part load code minimum efficiencies are not targeted at or intended to describe a single specific equipment model or brand but to represent the minimum performance of a group of equipment. Groups are generally defined by equipment size, application, and configuration (e.g., air- and water-cooled chillers). To the authors’ knowledge, there is currently no publicly available resources or methodology to generate such sets of performance curves, that is sets of metric specific generic performance curves for BES.
+
+For years, chillers have been required in many building energy codes to have both a minimum full (expressed as a EER, COP, or kW/ton value) and part load efficiency (expressed as an IPLV value) which make them a good candidate for developing a methodology to develop such a set of curves.
+
+## E-mail and  Conference Call Conclusions ##
+
+## Overview ##
+The proposed changes add a new dataset, `CodeCompliantEquipment.idf`, that includes chiller performance curves to be use to meet the minimum requirements in ASHRAE 90.1-2019 and its Appendix G. We propose that the same dataset be improved and supplemented at a later date with additional performance curves for other equipment types (for instance: IEER-rated DX equipment).
+
+## Approach ##
+Generating chiller performances curves meeting both full and part load efficiency numbers require solving a largely underdetermined system of equations. The number of unknowns depend on the type of curves, which mostly depend on the algorithm used by the software and used in models. It also means that an infinite number of solutions exists. One way to solve this issue is to turn it into a constrained problem and use a simple genetic algorithm (GA) with a well-defined objective or cost function to determine potential solutions with physical significance. A GA can have a short computational time, be scalable, versatile, find solutions to problems with large number of parameters to problems with multiple local optima so it is a good candidate for this task.
+
```

Comment:
Did you mean "generic"

... constrained problem and use a simple genetic algorithm (GA) ...